### PR TITLE
Issue #144: Updated timestamp to show submission time and day

### PR DESCRIPTION
**Pull request by xsihe**

### CODE REVIEW

File: src/Homepage/EntriesGroup.js [hunk 1]
```diff
@@ -23,10 +21,27 @@ function Overlay(props) {
...
+
+  let time = hh + ":" + mm + " " + period;
+
+  if (timeDifference >= HOURS_24 && timeDifference < (7 * HOURS_24)) {
+    time += " on " + days[then.getDay()];
+  }
+  else if (timeDifference >= (7 * HOURS_24)) {
+    time += " " + (timeDifference / HOURS_24).toFixed() + " days ago";
+  }
+
```
Comment (Siddharth11) [inline src/Homepage/EntriesGroup.js:42]: The website preview looks good, though the code can be made bit more readable. We have integrated [date-fns](https://date-fns.org/docs/Getting-Started) package in this project. It has some nice utility functions. Do you think any of those can be used here ?

Comment (xsihe) [reply src/Homepage/EntriesGroup.js:42]: Sure, I'll take a look at it.

**Comment by Siddharth11** [comment]
@xsihe looks good! Can you resolve the merge conflict? I'll merge it then

**Comment by arjun27** [comment]
guess we can also remove `humanize-duration` from package.json?

**Comment by Siddharth11** [comment]
yeah

**Comment by arjun27** [comment]
removing

**Pull request merged**